wazo_agid/modules/call_recording.py

```diff
@@ -9,24 +9,34 @@
 
 
 def call_recording(agi, cursor, args):
+    calld = agi.config['calld']['client']
+    channel_id = agi.env['agi_uniqueid']
     if agi.get_variable('WAZO_CALL_RECORD_ACTIVE') == '1':
```

Comment:
Is it useful to keep that check if the API is also checking it?

Reply:
How you would know the toggle to do?

Reply:
Oh yeah indeed :laughing: 
It would also be possible to call the API to check the status of the call but it would add a lot of overhead...